src/rpcConnectionPool.ts

```diff
@@ -1,18 +1,16 @@
-import { Memento } from "vscode";
-
 import { INeoRpcConnection, NeoRpcConnection } from "./neoRpcConnection";
 
 export class RpcConnectionPool {
 
     private readonly connections: Map<string, INeoRpcConnection> = new Map<string, INeoRpcConnection>();
 
-    constructor(private readonly globalState: Memento) {
+    constructor() {
```

Comment:
Are we still caching any state?

Reply:
Between vscode lifetimes, no (non-empty block list was the only thing persisted that long).  There is still an in-memory cache for (most) RPC calls.